pkg/gadgets/audit/seccomp/tracer/bpf/audit-seccomp.bpf.c

```diff
@@ -30,20 +24,17 @@ struct {
 	__uint(type, BPF_MAP_TYPE_PERF_EVENT_ARRAY);
 } events SEC(".maps");
 
-const volatile bool filter_by_mnt_ns = false;
-
 SEC("kprobe/audit_seccomp")
 int ig_audit_secc(struct pt_regs *ctx)
 {
 	unsigned long syscall = PT_REGS_PARM1(ctx);
 	int code = PT_REGS_PARM3(ctx);
 
-	struct task_struct *task = (struct task_struct *)bpf_get_current_task();
-	__u64 mntns_id = BPF_CORE_READ(task, nsproxy, mnt_ns, ns.inum);
+	__u64 mntns_id = gadget_get_mntns_id();
 	if (mntns_id == 0)
 		return 0;
```

Comment:
I have no idea. It has been there since the gadget was implemented (by you :stuck_out_tongue:) -> https://github.com/inspektor-gadget/inspektor-gadget/commit/3149914b63d9370effbd35f7e935e14ecec5657c#diff-b2c17807374a77fa42b75006c9b44fb1b974f596ac19278ba43d17426146a621R37

IMO we can remove it but I'll prefer to make this on another PR.